[Docs] Update Pocket TestNet nodes

Olshansk opened the pull request:
1. Remove Nodefleet as the TestNet maintainer
2. Add cryptonodes & stakenodes as TestNet Maintainer

Olshansk committed:
WIP - pocket core TestNet nodes

Olshansk committed:
Update TestNet Docs

Olshansk committed:
Add Ian's endpoint

breezytm reviewed (commented):
The following sections need to be removed 
```Validator List w/ Pocket RPC```
```Validators w/ Tendermint RPC```

The following sections need to be updated to remove the extra `/` in the examples after the load balancer URLs
```curl https://tendermint.testnet.pokt.network/lb/6d6f727365//net_info```
```curl https://tendermint.testnet.pokt.network/lb/6d6f727365//status```

The `TestNet Seeds` list needs to be updated to reflect the following 
```b3d86cd8ab4aa0cb9861cb795d8d154e685a94cf@seed1.testnet.pokt.network:26663
5b0107a5252f6a037eed7f5c24a7d916e4dd93bd@testnet_seed_4.cryptonode.tools:16646```

Olshansk committed:
Update docs

Olshansk commented:
@breezytm Updated the docs. 

Thanks @breezytm

breezytm committed:
Add Ian's RPC endpoint

Ian posted his endpoint URL in Discord message: https://discord.com/channels/553741558869131266/1290707245872451625/1291598128847519815

Olshansk commented on doc/guides/testnet.md:
```diff
@@ -2,178 +2,63 @@
+### BreezyTm (StakeNodes)
 
-### Example Queries
+`breezytm | Stakenodes (277262895459336194)` has made his Validator's RPC + Tendermint
```
```suggestion
`breezytm | Stakenodes (277262895459336194)` Validator RPC + Tendermint
```

Olshansk commented on doc/guides/testnet.md:
```diff
@@ -2,178 +2,63 @@
+### Ian (Cryptonode.tools)
 
-Output:
+`Ian | cryptonode.tools (693644362575511573)` also made his Validator's endpoints available here
```
```suggestion
`Ian | cryptonode.tools (693644362575511573)` Validator endpoints available here:
```

Olshansk committed:
Update doc/guides/testnet.md

Olshansk committed:
Update doc/guides/testnet.md
